transaction.go

```diff
@@ -157,3 +157,8 @@ func (t *Transaction) syntacticallyValidate(protoParams *ProtocolParameters) err
 
 	return nil
 }
+
+func (t *Transaction) WorkScore(workScoreStructure *WorkScoreStructure) WorkScore {
+	return t.Unlocks.WorkScore(workScoreStructure) +
```

Comment:
The FactorData for `util.NumByteLen(uint32(PayloadTransaction))` is missing here.

Reply:
Added